hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/util/UtilityTest.java

```diff
@@ -131,4 +146,82 @@ void instantToTimestamp(long seconds, int nanos) {
     void toSnakeCase(String input, String output) {
         assertThat(Utility.toSnakeCase(input)).isEqualTo(output);
     }
+
+    @ParameterizedTest(name = "Log {0} with cause {2}")
+    @CsvSource(
+            textBlock =
+                    """
+                                plain message, plain message, false,
+                                {} arg message, one arg message, true, one
+                                {} {} message, a b message, false, 'a, b'
+                                {} {} {} {}, a b c d, true, 'a, b, c, d'
```

Comment:
It might be cleaner and easier to understand if the exception was moved into the args and the expected message. So that you just have `String format, String[] args, String expected` parameters. The converter could detect the string `exception` and turn it into a RuntimeException.

```suggestion
                                plain message, '', plain message
                                {} arg message, 'one,exception', one arg message: RuntimeException: exception
                                {} {} message, 'a,b', a b message
                                {} {} {} {}, 'a,b,c,d,exception', a b c d: RuntimeException: exception
```

Reply:
Revamped this test case, and the converter significantly. I like it much better. 👍